subprojects/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/MavenConversionIntegrationTest.groovy

```diff
@@ -332,6 +332,17 @@ Root project 'webinar-parent'
 """
     }
 
+    def "kotlin dsl is not supported"() {
+        given:
+        resources.maybeCopy('MavenConversionIntegrationTest/singleModule')
+
+        when:
+        fails 'init', '--dsl', 'kotlin'
+
+        then:
+        failure.assertHasCause("The requested DSL 'kotlin' is not supported in 'pom' setup type")
```

Comment:
❓ The requested DSL 'kotlin' is not supported by the setup type 'pom'

Reply:
This message is worded like the other existing ones about supported features.

Reply:
Ok.